frontend/src/v4/routes/viewerGui/components/sequences/sequences.component.tsx

```diff
@@ -126,21 +122,6 @@ export class Sequences extends PureComponent<IProps, {}> {
 		return <SequencesIcon />;
 	}
 
-	public onPlayStarted = () => {
```

Comment:
Why did you get rid of this?

Reply:
Because that logic was being called only when the play started, which was not quite accurate.
Imagine this scenario: you have just selected a view that has some transformations and some colour overrides. You then open a sequence and move the slider up and down. The colour overrides from the sequence are visible, however, the transformations and the transparencies don't, at least, until you hit play the first time.

That's not correct, as every time you change the sequence using the slider, you should see only the current view from the sequence. So I moved it in `sequences.sagas.tsx`, inside  `updateSelectedStateDefinition`. But what's really needed, is to clear the viewpoint, so what I put there is simply
```ts
if (!viewpoint) {
	yield put(ViewpointsActions.setActiveViewpoint(null));
}
```

Reply:
Its a different code is `deselectViewsAndLeaveClipping` I dont remember why it was using that (you'll need to check the  commits) but there was a reason for it

Reply:
Its a different code is deselectViewsAndLeaveClipping I dont remember why it was using that (you'll need to check the commits) but there was a reason for it

Reply:
It was introduced here https://github.com/3drepo/3drepo.io/pull/2247/commits/70bfc4bd1e7ebea5f6da68af48a63c1dd9342885 for this reason: ![image](https://github.com/3drepo/3drepo.io/assets/43547703/d3ac3646-3af2-44a0-8eb6-3c6333496102)

It seems correct then, what do you think?

Reply:
Yeah but take a look that it says leave clipping while the other one wont

Reply:
Yes, but resetting the viewpoint that way doesn't get rid of the clipping planes, so the final outcome in that sense is the same